libs/execution_context/ebpf_native.c

```diff
@@ -1003,14 +1040,24 @@ ebpf_native_initiate()
     return_value = ebpf_hash_table_create(
         &_ebpf_native_authorized_module_table,
         &(const ebpf_hash_table_creation_options_t){
-            .key_size = EBPF_SHA256_HASH_LENGTH,
-            .value_size = sizeof(uint32_t),
+            .key_size = sizeof(GUID),
+            .value_size = sizeof(ebpf_native_authorized_module_entry_t),
             .allocate = ebpf_allocate,
             .free = ebpf_free,
         });
     if (return_value != EBPF_SUCCESS) {
         goto Done;
     }
+    authorized_module_table_created = true;
+
+    return_value = ebpf_allocate_timer_work_item(
+        &_ebpf_native_authorized_module_cleanup_work_item,
+        _ebpf_native_authorized_module_cleanup_work_item_callback,
+        NULL);
```

Comment:
question: do we have plan to move the whole flow for loading native modules eventually to ebpfsvc? If yes, will we still need this timer?